doc/changelog.rst

```diff
@@ -12,12 +12,14 @@ Usability improvements/Changes
 - Adds `typing <https://docs.python.org/3/library/typing.html>`_ to user-facing functions
 - Adds :code:`autode.config.location` to easily locate the core configuration file for permanent editing
 - Updates documentation for readability
+- Ensures units are kept when :code:`Value(x)`, when :code:`x` is a :code:`Value`
```

Comment:
not sure this makes sense

Reply:
I know it doesn't. @tristan-j-wood  and I couldn't come up with anything much better. Have changed to something _hopefully_ easier to understand